moviebot/controller/controller_flask.py

```diff
@@ -0,0 +1,398 @@
+            {"payload": "/restart", "action": self.restart},
+            {"payload": "/exit", "action": self.exit},
+            {"payload": "/delete", "action": self.delete_data},
+            {"payload": "/test", "action": self.privacy_policy},
+            {"payload": "/store", "action": self.store_user},
+            {"payload": "/continue", "action": self.start_conversation},
+        ]
+        self.agent_intent = ""
+
+    def lookup(self) -> sqlite3.Cursor:
+        """Returns SQL cursor."""
+        conn = sqlite3.connect(self.configuration["DATA"]["db_path"])
+        c = conn.cursor()
+        return c
+
+    def initialize(self, user_id: str) -> None:
+        """Initializes structs for a new user.
+
+        Args:
+            user_id: User id.
+        """
+        if user_id not in self.agent:
+            self.user_options[user_id] = {}
+            self.users[user_id] = {}
+            self.user_messages[user_id] = HTTPDataFormatter(user_id)
+            self.start_agent(user_id)
+
+    def start_conversation(self, user_id: str) -> HTTP_OBJECT_MESSAGE:
+        """Starts conversation with agent and sends instructions to user.
+
+        Args:
+            user_id: User id.
+
+        Returns:
+            Object with start message to send to the server.
+        """
+        start_text = self.start_agent(user_id)
+        instructions_text = self.instructions(user_id)
+        start_text["message"]["text"] = "\n\n".join(
+            [
+                start_text["message"]["text"],
+                instructions_text["message"]["text"],
+            ]
+        )
+        return start_text
+
+    def movie_info(self, movie_id: str, user_id: str) -> None:
+        """Retrieves relevant movie info from database for selected user.
+
+        Args:
+            movie_id: Movie id.
+            user_id: User id.
+        """
+        for row in self.lookup().execute(
+            f'SELECT * FROM movies_v2 WHERE ID="{movie_id}"'
+        ):
+            self.info[user_id] = {
+                "title": row[1],
+                "rating": row[4],
+                "duration": row[6],
+                "summary": row[10],
+                "image_url": row[9],
+                "imdb_link": row[12],
+            }
+
+    def execute_agent(self, configuration: Dict[str, Any]) -> None:
+        """Configures the controller.
+
+        Args:
+            configuration: Configuration for the agent.
+        """
+        self.configuration = configuration
+        self.configuration["new_user"] = {}
+        if self.configuration["BOT_HISTORY"]["path"]:
+            self.path = self.configuration["BOT_HISTORY"]["path"]
+
+    def restart(self, user_id: str) -> Callable:
+        """Restarts agent for this user."""
+        return self.start_agent(user_id, True)
+
+    def start_agent(self, user_id: str, restart=False) -> HTTP_OBJECT_MESSAGE:
+        """Starts conversation with agent.
```

Comment:
This docstring describes `start_conversation` method. Maybe `start_agent` should not output anything, just initialize?

Reply:
The function start_agent was removed and the initialisation is done in initialize.